{
    repas_par_jour: {
      "2024-01-01": {
        petit_dejeuner: "Nom de la recette",
        dejeuner: "Nom de la recette",
        diner: "Nom de la recette"
      },
      // ... autres jours
    }
  }[05/Nov/2024 02:40:13] INFO autoreload Watching for file changes with StatReloader
[05/Nov/2024 02:41:12] INFO autoreload /home/dafa/gestion-recettes/Backend/recettes/views.py changed, reloading.
[05/Nov/2024 02:41:13] INFO autoreload Watching for file changes with StatReloader
[05/Nov/2024 02:41:22] INFO views Début de la génération du plan repas - Données reçues: {'utilisateur': '7', 'date_debut': '2024-11-11', 'date_fin': '2024-11-17'}
[05/Nov/2024 02:41:22] INFO views Paramètres : utilisateur=7, date_debut=2024-11-11, date_fin=2024-11-17
[05/Nov/2024 02:41:22] INFO views Recherche des préférences pour l'utilisateur 7
[05/Nov/2024 02:41:22] INFO views Préférences trouvées - Végétarien: True
[05/Nov/2024 02:41:22] INFO views Récupération des recettes
[05/Nov/2024 02:41:22] ERROR views Erreur lors de la récupération des recettes: 
Traceback (most recent call last):
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/query.py", line 857, in parse
    return handler(self, statement)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/query.py", line 933, in _select
    return SelectQuery(self.db, self.connection_properties, sm, self._params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/query.py", line 116, in __init__
    super().__init__(*args)
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/query.py", line 62, in __init__
    self.parse()
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/query.py", line 152, in parse
    self.where = WhereConverter(self, statement)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/converters.py", line 27, in __init__
    self.parse()
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/converters.py", line 119, in parse
    self.op = WhereOp(
              ^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/operators.py", line 475, in __init__
    self._statement2ops()
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/operators.py", line 438, in _statement2ops
    if prev_op.lhs is None:
       ^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'lhs'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/cursor.py", line 51, in execute
    self.result = Query(
                  ^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/query.py", line 784, in __init__
    self._query = self.parse()
                  ^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/query.py", line 885, in parse
    raise exe from e
djongo.exceptions.SQLDecodeError: 

	Keyword: None
	Sub SQL: None
	FAILED SQL: SELECT COUNT(*) AS "__count" FROM "recettes_recette" WHERE "recettes_recette"."vegetarien"
	Params: ()
	Version: 1.3.6

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/cursor.py", line 59, in execute
    raise db_exe from e
djongo.database.DatabaseError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/dafa/gestion-recettes/Backend/recettes/views.py", line 463, in create
    logger.info(f"Nombre de recettes végétariennes trouvées : {recettes.count()}")
                                                               ^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/models/query.py", line 621, in count
    return self.query.get_count(using=self.db)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/models/sql/query.py", line 559, in get_count
    return obj.get_aggregation(using, ["__count"])["__count"]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/models/sql/query.py", line 544, in get_aggregation
    result = compiler.execute_sql(SINGLE)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/models/sql/compiler.py", line 1398, in execute_sql
    cursor.execute(sql, params)
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 102, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/cursor.py", line 59, in execute
    raise db_exe from e
django.db.utils.DatabaseError
[05/Nov/2024 02:41:23] ERROR views Erreur inattendue: 
Traceback (most recent call last):
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/query.py", line 857, in parse
    return handler(self, statement)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/query.py", line 933, in _select
    return SelectQuery(self.db, self.connection_properties, sm, self._params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/query.py", line 116, in __init__
    super().__init__(*args)
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/query.py", line 62, in __init__
    self.parse()
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/query.py", line 152, in parse
    self.where = WhereConverter(self, statement)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/converters.py", line 27, in __init__
    self.parse()
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/converters.py", line 119, in parse
    self.op = WhereOp(
              ^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/operators.py", line 475, in __init__
    self._statement2ops()
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/operators.py", line 438, in _statement2ops
    if prev_op.lhs is None:
       ^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'lhs'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/cursor.py", line 51, in execute
    self.result = Query(
                  ^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/query.py", line 784, in __init__
    self._query = self.parse()
                  ^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/query.py", line 885, in parse
    raise exe from e
djongo.exceptions.SQLDecodeError: 

	Keyword: None
	Sub SQL: None
	FAILED SQL: SELECT COUNT(*) AS "__count" FROM "recettes_recette" WHERE "recettes_recette"."vegetarien"
	Params: ()
	Version: 1.3.6

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/cursor.py", line 59, in execute
    raise db_exe from e
djongo.database.DatabaseError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/dafa/gestion-recettes/Backend/recettes/views.py", line 463, in create
    logger.info(f"Nombre de recettes végétariennes trouvées : {recettes.count()}")
                                                               ^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/models/query.py", line 621, in count
    return self.query.get_count(using=self.db)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/models/sql/query.py", line 559, in get_count
    return obj.get_aggregation(using, ["__count"])["__count"]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/models/sql/query.py", line 544, in get_aggregation
    result = compiler.execute_sql(SINGLE)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/models/sql/compiler.py", line 1398, in execute_sql
    cursor.execute(sql, params)
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 102, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/cursor.py", line 59, in execute
    raise db_exe from e
django.db.utils.DatabaseError
[05/Nov/2024 02:41:23] ERROR log Internal Server Error: /api/plan_repas/
[05/Nov/2024 02:45:27] INFO autoreload /home/dafa/gestion-recettes/Backend/recettes/views.py changed, reloading.
[05/Nov/2024 02:45:27] INFO autoreload Watching for file changes with StatReloader
[05/Nov/2024 02:46:09] INFO views Début de la génération du plan repas - Données reçues: {'utilisateur': '7', 'date_debut': '2024-11-11', 'date_fin': '2024-11-17'}
[05/Nov/2024 02:46:09] INFO views Paramètres : utilisateur=7, date_debut=2024-11-11, date_fin=2024-11-17
[05/Nov/2024 02:46:09] INFO views Recherche des préférences pour l'utilisateur 7
[05/Nov/2024 02:46:09] INFO views Préférences trouvées - Végétarien: True
[05/Nov/2024 02:46:09] INFO views Récupération des recettes
[05/Nov/2024 02:46:09] ERROR views Erreur lors de la récupération des recettes: 
Traceback (most recent call last):
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/query.py", line 857, in parse
    return handler(self, statement)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/query.py", line 933, in _select
    return SelectQuery(self.db, self.connection_properties, sm, self._params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/query.py", line 116, in __init__
    super().__init__(*args)
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/query.py", line 62, in __init__
    self.parse()
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/query.py", line 152, in parse
    self.where = WhereConverter(self, statement)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/converters.py", line 27, in __init__
    self.parse()
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/converters.py", line 119, in parse
    self.op = WhereOp(
              ^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/operators.py", line 475, in __init__
    self._statement2ops()
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/operators.py", line 438, in _statement2ops
    if prev_op.lhs is None:
       ^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'lhs'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/cursor.py", line 51, in execute
    self.result = Query(
                  ^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/query.py", line 784, in __init__
    self._query = self.parse()
                  ^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/query.py", line 885, in parse
    raise exe from e
djongo.exceptions.SQLDecodeError: 

	Keyword: None
	Sub SQL: None
	FAILED SQL: SELECT "recettes_recette"."_id", "recettes_recette"."nom", "recettes_recette"."description", "recettes_recette"."instructions", "recettes_recette"."vegetarien", "recettes_recette"."categorie", "recettes_recette"."image" FROM "recettes_recette" WHERE "recettes_recette"."vegetarien"
	Params: ()
	Version: 1.3.6

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/cursor.py", line 59, in execute
    raise db_exe from e
djongo.database.DatabaseError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/dafa/gestion-recettes/Backend/recettes/views.py", line 468, in create
    toutes_recettes = list(recettes)
                      ^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/models/query.py", line 394, in __iter__
    self._fetch_all()
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/models/query.py", line 1867, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/models/query.py", line 87, in __iter__
    results = compiler.execute_sql(
              ^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/models/sql/compiler.py", line 1398, in execute_sql
    cursor.execute(sql, params)
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 102, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/cursor.py", line 59, in execute
    raise db_exe from e
django.db.utils.DatabaseError
[05/Nov/2024 02:46:09] ERROR views Erreur inattendue: 
Traceback (most recent call last):
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/query.py", line 857, in parse
    return handler(self, statement)
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/query.py", line 933, in _select
    return SelectQuery(self.db, self.connection_properties, sm, self._params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/query.py", line 116, in __init__
    super().__init__(*args)
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/query.py", line 62, in __init__
    self.parse()
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/query.py", line 152, in parse
    self.where = WhereConverter(self, statement)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/converters.py", line 27, in __init__
    self.parse()
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/converters.py", line 119, in parse
    self.op = WhereOp(
              ^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/operators.py", line 475, in __init__
    self._statement2ops()
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/operators.py", line 438, in _statement2ops
    if prev_op.lhs is None:
       ^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'lhs'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/cursor.py", line 51, in execute
    self.result = Query(
                  ^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/query.py", line 784, in __init__
    self._query = self.parse()
                  ^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/sql2mongo/query.py", line 885, in parse
    raise exe from e
djongo.exceptions.SQLDecodeError: 

	Keyword: None
	Sub SQL: None
	FAILED SQL: SELECT "recettes_recette"."_id", "recettes_recette"."nom", "recettes_recette"."description", "recettes_recette"."instructions", "recettes_recette"."vegetarien", "recettes_recette"."categorie", "recettes_recette"."image" FROM "recettes_recette" WHERE "recettes_recette"."vegetarien"
	Params: ()
	Version: 1.3.6

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/cursor.py", line 59, in execute
    raise db_exe from e
djongo.database.DatabaseError

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/dafa/gestion-recettes/Backend/recettes/views.py", line 468, in create
    toutes_recettes = list(recettes)
                      ^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/models/query.py", line 394, in __iter__
    self._fetch_all()
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/models/query.py", line 1867, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
                         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/models/query.py", line 87, in __iter__
    results = compiler.execute_sql(
              ^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/models/sql/compiler.py", line 1398, in execute_sql
    cursor.execute(sql, params)
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 102, in execute
    return super().execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 67, in execute
    return self._execute_with_wrappers(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/db/backends/utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/djongo/cursor.py", line 59, in execute
    raise db_exe from e
django.db.utils.DatabaseError
[05/Nov/2024 02:46:09] ERROR log Internal Server Error: /api/plan_repas/
[05/Nov/2024 02:53:19] INFO autoreload /home/dafa/gestion-recettes/Backend/recettes/views.py changed, reloading.
[05/Nov/2024 02:53:20] INFO autoreload Watching for file changes with StatReloader
[05/Nov/2024 02:53:26] INFO views Début de la génération du plan repas - Données reçues: {'utilisateur': 1, 'date_debut': '2024-11-11', 'date_fin': '2024-11-17'}
[05/Nov/2024 02:53:26] INFO views Paramètres : utilisateur=1, date_debut=2024-11-11, date_fin=2024-11-17
[05/Nov/2024 02:53:26] INFO views Recherche des préférences pour l'utilisateur 1
[05/Nov/2024 02:53:26] INFO views Préférences trouvées - Végétarien: True
[05/Nov/2024 02:53:26] INFO views Récupération des recettes
[05/Nov/2024 02:53:26] INFO views Nombre total de recettes trouvées : 8
[05/Nov/2024 02:53:26] INFO views Nombre de recettes végétariennes : 6
[05/Nov/2024 02:53:26] WARNING log Bad Request: /api/plan_repas/
[05/Nov/2024 02:53:57] INFO views Début de la génération du plan repas - Données reçues: {'utilisateur': '7', 'date_debut': '2024-11-11', 'date_fin': '2024-11-17'}
[05/Nov/2024 02:53:57] INFO views Paramètres : utilisateur=7, date_debut=2024-11-11, date_fin=2024-11-17
[05/Nov/2024 02:53:57] INFO views Recherche des préférences pour l'utilisateur 7
[05/Nov/2024 02:53:57] INFO views Préférences trouvées - Végétarien: False
[05/Nov/2024 02:53:57] INFO views Récupération des recettes
[05/Nov/2024 02:53:57] INFO views Nombre total de recettes trouvées : 8
[05/Nov/2024 02:54:24] INFO views Début de la génération du plan repas - Données reçues: {'utilisateur': '1', 'date_debut': '2024-11-11', 'date_fin': '2024-11-17'}
[05/Nov/2024 02:54:24] INFO views Paramètres : utilisateur=1, date_debut=2024-11-11, date_fin=2024-11-17
[05/Nov/2024 02:54:24] INFO views Recherche des préférences pour l'utilisateur 1
[05/Nov/2024 02:54:24] INFO views Préférences trouvées - Végétarien: True
[05/Nov/2024 02:54:24] INFO views Récupération des recettes
[05/Nov/2024 02:54:24] INFO views Nombre total de recettes trouvées : 8
[05/Nov/2024 02:54:24] INFO views Nombre de recettes végétariennes : 6
[05/Nov/2024 02:54:24] WARNING log Bad Request: /api/plan_repas/
[05/Nov/2024 02:54:33] INFO views Début de la génération du plan repas - Données reçues: {'utilisateur': '6', 'date_debut': '2024-11-11', 'date_fin': '2024-11-17'}
[05/Nov/2024 02:54:33] INFO views Paramètres : utilisateur=6, date_debut=2024-11-11, date_fin=2024-11-17
[05/Nov/2024 02:54:33] INFO views Recherche des préférences pour l'utilisateur 6
[05/Nov/2024 02:54:33] INFO views Préférences trouvées - Végétarien: False
[05/Nov/2024 02:54:33] INFO views Récupération des recettes
[05/Nov/2024 02:54:33] INFO views Nombre total de recettes trouvées : 8
[05/Nov/2024 02:58:42] INFO views Début de la génération du plan repas - Données reçues: {'utilisateur': '1', 'date_debut': '2024-11-11', 'date_fin': '2024-11-17'}
[05/Nov/2024 02:58:42] INFO views Paramètres : utilisateur=1, date_debut=2024-11-11, date_fin=2024-11-17
[05/Nov/2024 02:58:42] INFO views Recherche des préférences pour l'utilisateur 1
[05/Nov/2024 02:58:42] INFO views Préférences trouvées - Végétarien: True
[05/Nov/2024 02:58:42] INFO views Récupération des recettes
[05/Nov/2024 02:58:42] INFO views Nombre total de recettes trouvées : 8
[05/Nov/2024 02:58:42] INFO views Nombre de recettes végétariennes : 6
[05/Nov/2024 02:58:42] WARNING log Bad Request: /api/plan_repas/
[05/Nov/2024 03:00:04] INFO autoreload /home/dafa/gestion-recettes/Backend/recettes/views.py changed, reloading.
[05/Nov/2024 03:00:04] INFO autoreload Watching for file changes with StatReloader
[05/Nov/2024 03:00:10] INFO views Début de la génération du plan repas - Données reçues: {'utilisateur': 1, 'date_debut': '2024-11-11', 'date_fin': '2024-11-17'}
[05/Nov/2024 03:00:10] INFO views Paramètres : utilisateur=1, date_debut=2024-11-11, date_fin=2024-11-17
[05/Nov/2024 03:00:10] INFO views Recherche des préférences pour l'utilisateur 1
[05/Nov/2024 03:00:10] INFO views Préférences trouvées - Végétarien: True
[05/Nov/2024 03:00:10] INFO views Récupération des recettes
[05/Nov/2024 03:00:10] INFO views Nombre total de recettes trouvées : 8
[05/Nov/2024 03:00:10] INFO views Nombre de recettes végétariennes : 6
[05/Nov/2024 03:00:10] INFO views Plan repas généré avec succès pour 7 jours à partir de 6 recettes disponibles
[05/Nov/2024 03:03:10] INFO views Début de la génération du plan repas - Données reçues: {'utilisateur': 1, 'date_debut': '2024-11-11', 'date_fin': '2024-11-17'}
[05/Nov/2024 03:03:10] INFO views Paramètres : utilisateur=1, date_debut=2024-11-11, date_fin=2024-11-17
[05/Nov/2024 03:03:10] INFO views Recherche des préférences pour l'utilisateur 1
[05/Nov/2024 03:03:10] INFO views Préférences trouvées - Végétarien: True
[05/Nov/2024 03:03:10] INFO views Récupération des recettes
[05/Nov/2024 03:03:10] INFO views Nombre total de recettes trouvées : 8
[05/Nov/2024 03:03:10] INFO views Nombre de recettes végétariennes : 6
[05/Nov/2024 03:03:10] INFO views Plan repas généré avec succès pour 7 jours à partir de 6 recettes disponibles
[05/Nov/2024 03:03:32] INFO views Début de la génération du plan repas - Données reçues: {'utilisateur': 1, 'date_debut': '2024-11-11', 'date_fin': '2024-11-17'}
[05/Nov/2024 03:03:32] INFO views Paramètres : utilisateur=1, date_debut=2024-11-11, date_fin=2024-11-17
[05/Nov/2024 03:03:32] INFO views Recherche des préférences pour l'utilisateur 1
[05/Nov/2024 03:03:32] INFO views Préférences trouvées - Végétarien: True
[05/Nov/2024 03:03:32] INFO views Récupération des recettes
[05/Nov/2024 03:03:32] INFO views Nombre total de recettes trouvées : 8
[05/Nov/2024 03:03:32] INFO views Nombre de recettes végétariennes : 6
[05/Nov/2024 03:03:32] INFO views Plan repas généré avec succès pour 7 jours à partir de 6 recettes disponibles
[05/Nov/2024 03:08:30] INFO autoreload /home/dafa/gestion-recettes/Backend/recettes/views.py changed, reloading.
[05/Nov/2024 03:08:31] INFO autoreload Watching for file changes with StatReloader
[05/Nov/2024 03:08:39] INFO views Début de la génération du plan repas - Données reçues: {'utilisateur': '7', 'date_debut': '2024-11-11', 'date_fin': '2024-11-17'}
[05/Nov/2024 03:08:39] INFO views Paramètres : utilisateur=7, date_debut=2024-11-11, date_fin=2024-11-17
[05/Nov/2024 03:08:39] INFO views Recherche des préférences pour l'utilisateur 7
[05/Nov/2024 03:08:39] INFO views Préférences trouvées - Végétarien: False
[05/Nov/2024 03:08:39] INFO views Récupération des recettes
[05/Nov/2024 03:08:39] INFO views Nombre total de recettes trouvées : 8
[05/Nov/2024 03:08:39] INFO views Plan repas généré avec succès pour 7 jours (21 repas) à partir de 8 recettes disponibles
[05/Nov/2024 03:11:36] INFO views Début de la génération du plan repas - Données reçues: {'utilisateur': 1, 'date_debut': '2024-11-11', 'date_fin': '2024-11-17'}
[05/Nov/2024 03:11:36] INFO views Paramètres : utilisateur=1, date_debut=2024-11-11, date_fin=2024-11-17
[05/Nov/2024 03:11:36] INFO views Recherche des préférences pour l'utilisateur 1
[05/Nov/2024 03:11:36] INFO views Préférences trouvées - Végétarien: True
[05/Nov/2024 03:11:36] INFO views Récupération des recettes
[05/Nov/2024 03:11:36] INFO views Nombre total de recettes trouvées : 8
[05/Nov/2024 03:11:36] INFO views Nombre de recettes végétariennes : 6
[05/Nov/2024 03:11:36] INFO views Plan repas généré avec succès pour 7 jours (21 repas) à partir de 6 recettes disponibles
[05/Nov/2024 03:41:05] INFO autoreload /home/dafa/gestion-recettes/Backend/recettes/views.py changed, reloading.
[05/Nov/2024 03:41:06] INFO autoreload Watching for file changes with StatReloader
[05/Nov/2024 03:41:09] WARNING log Unauthorized: /api/utilisateurs/
[05/Nov/2024 03:41:09] WARNING log Unauthorized: /api/utilisateurs/
[05/Nov/2024 03:41:25] INFO views Tentative de connexion pour l'utilisateur: lala
[05/Nov/2024 03:41:25] INFO views Données de la requête: {'username': 'lala', 'password': '1234567'}
[05/Nov/2024 03:41:25] INFO views Utilisateur lala authentifié avec succès
[05/Nov/2024 03:41:37] INFO views Début de la génération du plan repas - Données reçues: {'utilisateur': '7', 'date_debut': '2024-11-11', 'date_fin': '2024-11-17'}
[05/Nov/2024 03:41:37] INFO views Paramètres : utilisateur=7, date_debut=2024-11-11, date_fin=2024-11-17
[05/Nov/2024 03:41:37] INFO views Recherche des préférences pour l'utilisateur 7
[05/Nov/2024 03:41:37] INFO views Préférences trouvées - Végétarien: False
[05/Nov/2024 03:41:37] INFO views Récupération des recettes
[05/Nov/2024 03:41:37] INFO views Nombre total de recettes trouvées : 8
[05/Nov/2024 03:41:37] INFO views Plan repas généré avec succès pour 7 jours (21 repas) à partir de 8 recettes disponibles
[05/Nov/2024 03:51:09] INFO autoreload Watching for file changes with StatReloader
[05/Nov/2024 03:59:47] INFO views Tentative de connexion pour l'utilisateur: lala
[05/Nov/2024 03:59:47] INFO views Données de la requête: {'username': 'lala', 'password': '1234567'}
[05/Nov/2024 03:59:48] INFO views Utilisateur lala authentifié avec succès
[05/Nov/2024 04:00:43] INFO autoreload /home/dafa/gestion-recettes/Backend/utilisateurs/views.py changed, reloading.
[05/Nov/2024 04:00:44] INFO autoreload Watching for file changes with StatReloader
[05/Nov/2024 04:01:33] INFO autoreload /home/dafa/gestion-recettes/Backend/utilisateurs/views.py changed, reloading.
[05/Nov/2024 04:01:34] INFO autoreload Watching for file changes with StatReloader
[05/Nov/2024 04:01:45] INFO autoreload /home/dafa/gestion-recettes/Backend/recettes/views.py changed, reloading.
[05/Nov/2024 04:01:46] INFO autoreload Watching for file changes with StatReloader
[05/Nov/2024 04:02:00] INFO views Début de la génération du plan repas - Données reçues: {'utilisateur': '7', 'date_debut': '2024-11-11', 'date_fin': '2024-11-17'}
[05/Nov/2024 04:02:00] INFO views Paramètres : utilisateur=7, date_debut=2024-11-11, date_fin=2024-11-17
[05/Nov/2024 04:02:00] INFO views Recherche des préférences pour l'utilisateur 7
[05/Nov/2024 04:02:00] INFO views Préférences trouvées - Végétarien: False, Allergies: ['pomme']
[05/Nov/2024 04:02:00] INFO views Récupération des recettes
[05/Nov/2024 04:02:00] INFO views Nombre total de recettes trouvées : 8
[05/Nov/2024 04:02:00] INFO views Nombre de recettes après filtrage (végétarien et allergies) : 7
[05/Nov/2024 04:02:00] INFO views Plan repas généré avec succès pour 7 jours (21 repas) à partir de 7 recettes disponibles
[05/Nov/2024 04:03:18] INFO views Nombre de recettes récupérées: 8
[05/Nov/2024 04:03:18] INFO views Nombre de recettes récupérées: 8
[05/Nov/2024 04:07:54] INFO views Nombre de recettes récupérées: 8
[05/Nov/2024 04:07:55] INFO views Nombre de recettes récupérées: 8
[05/Nov/2024 04:09:27] INFO views Recette créée avec succès: Ratatouille Provençale
[05/Nov/2024 04:09:28] INFO views Nombre de recettes récupérées: 9
[05/Nov/2024 04:09:28] INFO views Nombre de recettes récupérées: 9
[05/Nov/2024 04:12:15] INFO views Recette créée avec succès: Coq au Vin
[05/Nov/2024 04:12:16] INFO views Nombre de recettes récupérées: 10
[05/Nov/2024 04:12:17] INFO views Nombre de recettes récupérées: 10
[05/Nov/2024 04:14:51] INFO views Recette créée avec succès: Salade Niçoise
[05/Nov/2024 04:14:51] INFO views Nombre de recettes récupérées: 11
[05/Nov/2024 04:14:52] INFO views Nombre de recettes récupérées: 11
[05/Nov/2024 04:17:22] INFO views Recette créée avec succès: Tarte aux Champignons
[05/Nov/2024 04:17:23] INFO views Nombre de recettes récupérées: 12
[05/Nov/2024 04:17:23] INFO views Nombre de recettes récupérées: 12
[05/Nov/2024 04:18:02] INFO views Nombre de recettes récupérées: 12
[05/Nov/2024 04:18:03] INFO views Nombre de recettes récupérées: 12
[05/Nov/2024 04:20:08] INFO views Recette créée avec succès: Crème Brûlée
[05/Nov/2024 04:20:09] INFO views Nombre de recettes récupérées: 13
[05/Nov/2024 04:20:09] INFO views Nombre de recettes récupérées: 13
[05/Nov/2024 04:20:34] INFO views Nombre de recettes récupérées: 13
[05/Nov/2024 04:20:35] INFO views Nombre de recettes récupérées: 13
[05/Nov/2024 04:20:46] INFO views Nombre de recettes récupérées: 13
[05/Nov/2024 04:20:46] INFO views Nombre de recettes récupérées: 13
[05/Nov/2024 04:23:12] INFO views Recette créée avec succès: Blanquette de Veau
[05/Nov/2024 04:23:12] INFO views Nombre de recettes récupérées: 14
[05/Nov/2024 04:23:13] INFO views Nombre de recettes récupérées: 14
[05/Nov/2024 04:25:29] INFO views Recette créée avec succès: Soupe à l'Oignon
[05/Nov/2024 04:25:29] INFO views Nombre de recettes récupérées: 15
[05/Nov/2024 04:25:30] INFO views Nombre de recettes récupérées: 15
[05/Nov/2024 04:28:22] INFO views Recette créée avec succès: Velouté de Potimarron
[05/Nov/2024 04:28:22] INFO views Nombre de recettes récupérées: 16
[05/Nov/2024 04:28:23] INFO views Nombre de recettes récupérées: 16
[05/Nov/2024 04:31:10] INFO views Recette créée avec succès: Risotto aux Champignons
[05/Nov/2024 04:31:10] INFO views Nombre de recettes récupérées: 17
[05/Nov/2024 04:31:11] INFO views Nombre de recettes récupérées: 17
[05/Nov/2024 04:33:00] INFO views Recette créée avec succès: Tartiflette Savoyarde
[05/Nov/2024 04:33:00] INFO views Nombre de recettes récupérées: 18
[05/Nov/2024 04:33:01] INFO views Nombre de recettes récupérées: 18
[05/Nov/2024 04:35:41] INFO views Nombre de recettes récupérées: 18
[05/Nov/2024 04:35:41] INFO views Nombre de recettes récupérées: 18
[05/Nov/2024 04:36:01] INFO views Début de la génération du plan repas - Données reçues: {'utilisateur': '7', 'date_debut': '2024-11-11', 'date_fin': '2024-11-17'}
[05/Nov/2024 04:36:01] INFO views Paramètres : utilisateur=7, date_debut=2024-11-11, date_fin=2024-11-17
[05/Nov/2024 04:36:01] INFO views Recherche des préférences pour l'utilisateur 7
[05/Nov/2024 04:36:01] INFO views Préférences trouvées - Végétarien: False, Allergies: ['pomme']
[05/Nov/2024 04:36:01] INFO views Récupération des recettes
[05/Nov/2024 04:36:01] INFO views Nombre total de recettes trouvées : 18
[05/Nov/2024 04:36:01] INFO views Nombre de recettes après filtrage (végétarien et allergies) : 17
[05/Nov/2024 04:36:01] INFO views Plan repas généré avec succès pour 7 jours (21 repas) à partir de 17 recettes disponibles
[05/Nov/2024 04:37:01] INFO views Début de la génération du plan repas - Données reçues: {'utilisateur': '7', 'date_debut': '2024-11-11', 'date_fin': '2024-11-17'}
[05/Nov/2024 04:37:01] INFO views Paramètres : utilisateur=7, date_debut=2024-11-11, date_fin=2024-11-17
[05/Nov/2024 04:37:01] INFO views Recherche des préférences pour l'utilisateur 7
[05/Nov/2024 04:37:01] INFO views Préférences trouvées - Végétarien: False, Allergies: ['pomme;chocolat']
[05/Nov/2024 04:37:01] INFO views Récupération des recettes
[05/Nov/2024 04:37:01] INFO views Nombre total de recettes trouvées : 18
[05/Nov/2024 04:37:01] INFO views Nombre de recettes après filtrage (végétarien et allergies) : 18
[05/Nov/2024 04:37:01] INFO views Plan repas généré avec succès pour 7 jours (21 repas) à partir de 18 recettes disponibles
[05/Nov/2024 04:37:07] INFO views Nombre de recettes récupérées: 18
[05/Nov/2024 04:37:07] INFO views Nombre de recettes récupérées: 18
[05/Nov/2024 04:37:18] INFO views Nombre de recettes récupérées: 18
[05/Nov/2024 04:37:18] INFO views Nombre de recettes récupérées: 18
[05/Nov/2024 04:37:37] INFO views Nombre de recettes récupérées: 18
[05/Nov/2024 04:37:38] INFO views Nombre de recettes récupérées: 18
[05/Nov/2024 04:38:00] INFO views Début de la génération du plan repas - Données reçues: {'utilisateur': '7', 'date_debut': '2024-11-11', 'date_fin': '2024-11-17'}
[05/Nov/2024 04:38:00] INFO views Paramètres : utilisateur=7, date_debut=2024-11-11, date_fin=2024-11-17
[05/Nov/2024 04:38:00] INFO views Recherche des préférences pour l'utilisateur 7
[05/Nov/2024 04:38:00] INFO views Préférences trouvées - Végétarien: False, Allergies: ['pomme;chocolat']
[05/Nov/2024 04:38:00] INFO views Récupération des recettes
[05/Nov/2024 04:38:00] INFO views Nombre total de recettes trouvées : 18
[05/Nov/2024 04:38:00] INFO views Nombre de recettes après filtrage (végétarien et allergies) : 18
[05/Nov/2024 04:38:00] INFO views Plan repas généré avec succès pour 7 jours (21 repas) à partir de 18 recettes disponibles
[05/Nov/2024 04:38:22] INFO views Début de la génération du plan repas - Données reçues: {'utilisateur': '7', 'date_debut': '2024-11-11', 'date_fin': '2024-11-17'}
[05/Nov/2024 04:38:22] INFO views Paramètres : utilisateur=7, date_debut=2024-11-11, date_fin=2024-11-17
[05/Nov/2024 04:38:22] INFO views Recherche des préférences pour l'utilisateur 7
[05/Nov/2024 04:38:22] INFO views Préférences trouvées - Végétarien: False, Allergies: ['pomme;chocolat']
[05/Nov/2024 04:38:22] INFO views Récupération des recettes
[05/Nov/2024 04:38:22] INFO views Nombre total de recettes trouvées : 18
[05/Nov/2024 04:38:22] INFO views Nombre de recettes après filtrage (végétarien et allergies) : 18
[05/Nov/2024 04:38:22] INFO views Plan repas généré avec succès pour 7 jours (21 repas) à partir de 18 recettes disponibles
[05/Nov/2024 04:38:28] INFO views Début de la génération du plan repas - Données reçues: {'utilisateur': '7', 'date_debut': '2024-11-11', 'date_fin': '2024-11-17'}
[05/Nov/2024 04:38:28] INFO views Paramètres : utilisateur=7, date_debut=2024-11-11, date_fin=2024-11-17
[05/Nov/2024 04:38:28] INFO views Recherche des préférences pour l'utilisateur 7
[05/Nov/2024 04:38:28] INFO views Préférences trouvées - Végétarien: False, Allergies: ['pomme;chocolat']
[05/Nov/2024 04:38:28] INFO views Récupération des recettes
[05/Nov/2024 04:38:29] INFO views Nombre total de recettes trouvées : 18
[05/Nov/2024 04:38:29] INFO views Nombre de recettes après filtrage (végétarien et allergies) : 18
[05/Nov/2024 04:38:29] INFO views Plan repas généré avec succès pour 7 jours (21 repas) à partir de 18 recettes disponibles
[05/Nov/2024 04:38:52] INFO views Début de la génération du plan repas - Données reçues: {'utilisateur': '7', 'date_debut': '2024-11-11', 'date_fin': '2024-11-17'}
[05/Nov/2024 04:38:52] INFO views Paramètres : utilisateur=7, date_debut=2024-11-11, date_fin=2024-11-17
[05/Nov/2024 04:38:52] INFO views Recherche des préférences pour l'utilisateur 7
[05/Nov/2024 04:38:52] INFO views Préférences trouvées - Végétarien: False, Allergies: ['pomme']
[05/Nov/2024 04:38:52] INFO views Récupération des recettes
[05/Nov/2024 04:38:52] INFO views Nombre total de recettes trouvées : 18
[05/Nov/2024 04:38:52] INFO views Nombre de recettes après filtrage (végétarien et allergies) : 17
[05/Nov/2024 04:38:52] INFO views Plan repas généré avec succès pour 7 jours (21 repas) à partir de 17 recettes disponibles
[05/Nov/2024 04:38:59] INFO views Début de la génération du plan repas - Données reçues: {'utilisateur': '7', 'date_debut': '2024-11-11', 'date_fin': '2024-11-17'}
[05/Nov/2024 04:38:59] INFO views Paramètres : utilisateur=7, date_debut=2024-11-11, date_fin=2024-11-17
[05/Nov/2024 04:38:59] INFO views Recherche des préférences pour l'utilisateur 7
[05/Nov/2024 04:38:59] INFO views Préférences trouvées - Végétarien: False, Allergies: ['pomme']
[05/Nov/2024 04:38:59] INFO views Récupération des recettes
[05/Nov/2024 04:38:59] INFO views Nombre total de recettes trouvées : 18
[05/Nov/2024 04:38:59] INFO views Nombre de recettes après filtrage (végétarien et allergies) : 17
[05/Nov/2024 04:38:59] INFO views Plan repas généré avec succès pour 7 jours (21 repas) à partir de 17 recettes disponibles
[05/Nov/2024 04:41:45] INFO views Nombre de recettes récupérées: 18
[05/Nov/2024 04:41:45] INFO views Nombre de recettes récupérées: 18
[05/Nov/2024 04:41:47] INFO views Nombre de recettes récupérées: 18
[05/Nov/2024 04:41:48] INFO views Nombre de recettes récupérées: 18
[05/Nov/2024 04:43:50] INFO views Nombre de recettes récupérées: 18
[05/Nov/2024 04:43:50] INFO views Nombre de recettes récupérées: 18
[05/Nov/2024 04:50:40] INFO autoreload /home/dafa/gestion-recettes/Backend/recettes/views.py changed, reloading.
[05/Nov/2024 04:50:40] INFO autoreload Watching for file changes with StatReloader
[05/Nov/2024 04:50:45] WARNING log Unauthorized: /api/utilisateurs/
[05/Nov/2024 04:50:45] WARNING log Unauthorized: /api/utilisateurs/
[05/Nov/2024 04:50:46] WARNING log Unauthorized: /api/utilisateurs/
[05/Nov/2024 04:50:46] WARNING log Unauthorized: /api/utilisateurs/
[05/Nov/2024 04:50:58] INFO views Tentative de connexion pour l'utilisateur: lala
[05/Nov/2024 04:50:58] INFO views Données de la requête: {'username': 'lala', 'password': '1234567'}
[05/Nov/2024 04:50:59] INFO views Utilisateur lala authentifié avec succès
[05/Nov/2024 04:51:08] INFO views Début de la génération du plan repas - Données reçues: {'utilisateur': '7', 'date_debut': '2024-11-11', 'date_fin': '2024-11-17'}
[05/Nov/2024 04:51:08] INFO views Paramètres : utilisateur=7, date_debut=2024-11-11, date_fin=2024-11-17
[05/Nov/2024 04:51:08] INFO views Recherche des préférences pour l'utilisateur 7
[05/Nov/2024 04:51:08] INFO views Préférences trouvées - Végétarien: False, Allergies: ['pomme']
[05/Nov/2024 04:51:08] INFO views Récupération des recettes
[05/Nov/2024 04:51:08] INFO views Nombre total de recettes trouvées : 18
[05/Nov/2024 04:51:08] INFO views Nombre de recettes pour petit_dejeuner: 5
[05/Nov/2024 04:51:08] INFO views Nombre de recettes pour dejeuner: 5
[05/Nov/2024 04:51:08] INFO views Nombre de recettes pour diner: 8
[05/Nov/2024 04:51:08] INFO views Plan repas généré avec succès pour 7 jours
[05/Nov/2024 04:51:21] INFO views Début de la génération du plan repas - Données reçues: {'utilisateur': '7', 'date_debut': '2024-11-11', 'date_fin': '2024-11-17'}
[05/Nov/2024 04:51:21] INFO views Paramètres : utilisateur=7, date_debut=2024-11-11, date_fin=2024-11-17
[05/Nov/2024 04:51:21] INFO views Recherche des préférences pour l'utilisateur 7
[05/Nov/2024 04:51:21] INFO views Préférences trouvées - Végétarien: False, Allergies: ['pomme']
[05/Nov/2024 04:51:21] INFO views Récupération des recettes
[05/Nov/2024 04:51:22] INFO views Nombre total de recettes trouvées : 18
[05/Nov/2024 04:51:22] INFO views Nombre de recettes pour petit_dejeuner: 5
[05/Nov/2024 04:51:22] INFO views Nombre de recettes pour dejeuner: 5
[05/Nov/2024 04:51:22] INFO views Nombre de recettes pour diner: 8
[05/Nov/2024 04:51:22] INFO views Plan repas généré avec succès pour 7 jours
[05/Nov/2024 04:52:10] INFO views Début de la génération du plan repas - Données reçues: {'utilisateur': 1, 'date_debut': '2024-11-11', 'date_fin': '2024-11-17'}
[05/Nov/2024 04:52:10] INFO views Paramètres : utilisateur=1, date_debut=2024-11-11, date_fin=2024-11-17
[05/Nov/2024 04:52:10] INFO views Recherche des préférences pour l'utilisateur 1
[05/Nov/2024 04:52:10] INFO views Préférences trouvées - Végétarien: True, Allergies: []
[05/Nov/2024 04:52:10] INFO views Récupération des recettes
[05/Nov/2024 04:52:10] INFO views Nombre total de recettes trouvées : 18
[05/Nov/2024 04:52:10] INFO views Nombre de recettes pour petit_dejeuner: 5
[05/Nov/2024 04:52:10] INFO views Nombre de recettes pour dejeuner: 3
[05/Nov/2024 04:52:10] INFO views Nombre de recettes pour diner: 5
[05/Nov/2024 04:52:10] INFO views Plan repas généré avec succès pour 7 jours
[05/Nov/2024 04:52:25] INFO views Début de la génération du plan repas - Données reçues: {'utilisateur': 1, 'date_debut': '2024-11-11', 'date_fin': '2024-11-17'}
[05/Nov/2024 04:52:25] INFO views Paramètres : utilisateur=1, date_debut=2024-11-11, date_fin=2024-11-17
[05/Nov/2024 04:52:25] INFO views Recherche des préférences pour l'utilisateur 1
[05/Nov/2024 04:52:25] INFO views Préférences trouvées - Végétarien: True, Allergies: []
[05/Nov/2024 04:52:25] INFO views Récupération des recettes
[05/Nov/2024 04:52:25] INFO views Nombre total de recettes trouvées : 18
[05/Nov/2024 04:52:25] INFO views Nombre de recettes pour petit_dejeuner: 5
[05/Nov/2024 04:52:25] INFO views Nombre de recettes pour dejeuner: 3
[05/Nov/2024 04:52:25] INFO views Nombre de recettes pour diner: 5
[05/Nov/2024 04:52:25] INFO views Plan repas généré avec succès pour 7 jours
[05/Nov/2024 04:52:29] INFO views Début de la génération du plan repas - Données reçues: {'utilisateur': 1, 'date_debut': '2024-11-11', 'date_fin': '2024-11-17'}
[05/Nov/2024 04:52:29] INFO views Paramètres : utilisateur=1, date_debut=2024-11-11, date_fin=2024-11-17
[05/Nov/2024 04:52:29] INFO views Recherche des préférences pour l'utilisateur 1
[05/Nov/2024 04:52:29] INFO views Préférences trouvées - Végétarien: True, Allergies: []
[05/Nov/2024 04:52:29] INFO views Récupération des recettes
[05/Nov/2024 04:52:29] INFO views Nombre total de recettes trouvées : 18
[05/Nov/2024 04:52:29] INFO views Nombre de recettes pour petit_dejeuner: 5
[05/Nov/2024 04:52:29] INFO views Nombre de recettes pour dejeuner: 3
[05/Nov/2024 04:52:29] INFO views Nombre de recettes pour diner: 5
[05/Nov/2024 04:52:29] INFO views Plan repas généré avec succès pour 7 jours
[05/Nov/2024 05:05:45] INFO autoreload /home/dafa/gestion-recettes/Backend/recettes/views.py changed, reloading.
[05/Nov/2024 05:05:46] INFO autoreload Watching for file changes with StatReloader
[05/Nov/2024 05:05:53] INFO views Début de la génération du plan repas - Données reçues: {'utilisateur': '7', 'date_debut': '2024-11-11', 'date_fin': '2024-11-17'}
[05/Nov/2024 05:05:53] INFO views Paramètres : utilisateur=7, date_debut=2024-11-11, date_fin=2024-11-17
[05/Nov/2024 05:05:53] INFO views Recherche des préférences pour l'utilisateur 7
[05/Nov/2024 05:05:53] INFO views Préférences trouvées - Végétarien: False, Allergies: ['pomme']
[05/Nov/2024 05:05:53] INFO views Récupération des recettes
[05/Nov/2024 05:05:53] INFO views Nombre total de recettes trouvées : 18
[05/Nov/2024 05:05:53] INFO views Allergie 'pomme' trouvée dans la recette 'Tarte aux pommes'
[05/Nov/2024 05:05:53] INFO views Allergie 'pomme' trouvée dans la recette 'Tartiflette Savoyarde'
[05/Nov/2024 05:05:53] INFO views Nombre de recettes pour petit_dejeuner: 5
[05/Nov/2024 05:05:53] INFO views Nombre de recettes pour dejeuner: 5
[05/Nov/2024 05:05:53] INFO views Nombre de recettes pour diner: 8
[05/Nov/2024 05:05:53] INFO views Plan repas généré avec succès pour 7 jours
[05/Nov/2024 05:06:00] INFO views Début de la génération du plan repas - Données reçues: {'utilisateur': '7', 'date_debut': '2024-11-11', 'date_fin': '2024-11-17'}
[05/Nov/2024 05:06:00] INFO views Paramètres : utilisateur=7, date_debut=2024-11-11, date_fin=2024-11-17
[05/Nov/2024 05:06:00] INFO views Recherche des préférences pour l'utilisateur 7
[05/Nov/2024 05:06:00] INFO views Préférences trouvées - Végétarien: False, Allergies: ['pomme']
[05/Nov/2024 05:06:00] INFO views Récupération des recettes
[05/Nov/2024 05:06:01] INFO views Nombre total de recettes trouvées : 18
[05/Nov/2024 05:06:01] INFO views Allergie 'pomme' trouvée dans la recette 'Tarte aux pommes'
[05/Nov/2024 05:06:01] INFO views Allergie 'pomme' trouvée dans la recette 'Tartiflette Savoyarde'
[05/Nov/2024 05:06:01] INFO views Nombre de recettes pour petit_dejeuner: 5
[05/Nov/2024 05:06:01] INFO views Nombre de recettes pour dejeuner: 5
[05/Nov/2024 05:06:01] INFO views Nombre de recettes pour diner: 8
[05/Nov/2024 05:06:01] INFO views Plan repas généré avec succès pour 7 jours
[05/Nov/2024 05:06:02] INFO views Début de la génération du plan repas - Données reçues: {'utilisateur': '7', 'date_debut': '2024-11-11', 'date_fin': '2024-11-17'}
[05/Nov/2024 05:06:02] INFO views Paramètres : utilisateur=7, date_debut=2024-11-11, date_fin=2024-11-17
[05/Nov/2024 05:06:02] INFO views Recherche des préférences pour l'utilisateur 7
[05/Nov/2024 05:06:02] INFO views Préférences trouvées - Végétarien: False, Allergies: ['pomme']
[05/Nov/2024 05:06:02] INFO views Récupération des recettes
[05/Nov/2024 05:06:02] INFO views Nombre total de recettes trouvées : 18
[05/Nov/2024 05:06:02] INFO views Allergie 'pomme' trouvée dans la recette 'Tarte aux pommes'
[05/Nov/2024 05:06:02] INFO views Allergie 'pomme' trouvée dans la recette 'Tartiflette Savoyarde'
[05/Nov/2024 05:06:02] INFO views Nombre de recettes pour petit_dejeuner: 5
[05/Nov/2024 05:06:02] INFO views Nombre de recettes pour dejeuner: 5
[05/Nov/2024 05:06:02] INFO views Nombre de recettes pour diner: 8
[05/Nov/2024 05:06:02] INFO views Plan repas généré avec succès pour 7 jours
[05/Nov/2024 05:06:04] INFO views Début de la génération du plan repas - Données reçues: {'utilisateur': '7', 'date_debut': '2024-11-11', 'date_fin': '2024-11-17'}
[05/Nov/2024 05:06:04] INFO views Paramètres : utilisateur=7, date_debut=2024-11-11, date_fin=2024-11-17
[05/Nov/2024 05:06:04] INFO views Recherche des préférences pour l'utilisateur 7
[05/Nov/2024 05:06:04] INFO views Préférences trouvées - Végétarien: False, Allergies: ['pomme']
[05/Nov/2024 05:06:04] INFO views Récupération des recettes
[05/Nov/2024 05:06:04] INFO views Nombre total de recettes trouvées : 18
[05/Nov/2024 05:06:04] INFO views Allergie 'pomme' trouvée dans la recette 'Tarte aux pommes'
[05/Nov/2024 05:06:04] INFO views Allergie 'pomme' trouvée dans la recette 'Tartiflette Savoyarde'
[05/Nov/2024 05:06:04] INFO views Nombre de recettes pour petit_dejeuner: 5
[05/Nov/2024 05:06:04] INFO views Nombre de recettes pour dejeuner: 5
[05/Nov/2024 05:06:04] INFO views Nombre de recettes pour diner: 8
[05/Nov/2024 05:06:04] INFO views Plan repas généré avec succès pour 7 jours
[05/Nov/2024 05:06:05] INFO views Début de la génération du plan repas - Données reçues: {'utilisateur': '7', 'date_debut': '2024-11-11', 'date_fin': '2024-11-17'}
[05/Nov/2024 05:06:05] INFO views Paramètres : utilisateur=7, date_debut=2024-11-11, date_fin=2024-11-17
[05/Nov/2024 05:06:05] INFO views Recherche des préférences pour l'utilisateur 7
[05/Nov/2024 05:06:05] INFO views Préférences trouvées - Végétarien: False, Allergies: ['pomme']
[05/Nov/2024 05:06:05] INFO views Récupération des recettes
[05/Nov/2024 05:06:05] INFO views Nombre total de recettes trouvées : 18
[05/Nov/2024 05:06:05] INFO views Allergie 'pomme' trouvée dans la recette 'Tarte aux pommes'
[05/Nov/2024 05:06:05] INFO views Allergie 'pomme' trouvée dans la recette 'Tartiflette Savoyarde'
[05/Nov/2024 05:06:05] INFO views Nombre de recettes pour petit_dejeuner: 5
[05/Nov/2024 05:06:05] INFO views Nombre de recettes pour dejeuner: 5
[05/Nov/2024 05:06:05] INFO views Nombre de recettes pour diner: 8
[05/Nov/2024 05:06:05] INFO views Plan repas généré avec succès pour 7 jours
[05/Nov/2024 05:12:01] INFO autoreload /home/dafa/gestion-recettes/Backend/recettes/views.py changed, reloading.
[05/Nov/2024 05:12:01] INFO autoreload Watching for file changes with StatReloader
[05/Nov/2024 05:12:10] INFO views Début de la génération du plan repas - Données reçues: {'utilisateur': '7', 'date_debut': '2024-11-11', 'date_fin': '2024-11-17'}
[05/Nov/2024 05:12:10] INFO views Paramètres : utilisateur=7, date_debut=2024-11-11, date_fin=2024-11-17
[05/Nov/2024 05:12:10] INFO views Recherche des préférences pour l'utilisateur 7
[05/Nov/2024 05:12:10] INFO views Préférences trouvées - Végétarien: False, Allergies: ['pomme']
[05/Nov/2024 05:12:10] INFO views Récupération des recettes
[05/Nov/2024 05:12:10] INFO views Nombre total de recettes trouvées : 18
[05/Nov/2024 05:12:10] INFO views Allergie 'pomme' trouvée dans la recette 'Tarte aux pommes'
[05/Nov/2024 05:12:10] INFO views Allergie 'pomme' trouvée dans la recette 'Tartiflette Savoyarde'
[05/Nov/2024 05:12:10] INFO views Nombre de recettes pour petit_dejeuner: 5
[05/Nov/2024 05:12:10] INFO views Nombre de recettes pour dejeuner: 5
[05/Nov/2024 05:12:10] INFO views Nombre de recettes pour diner: 8
[05/Nov/2024 05:12:10] INFO views Plan repas généré avec succès pour 7 jours
[05/Nov/2024 05:28:52] INFO autoreload /home/dafa/gestion-recettes/Backend/recettes/views.py changed, reloading.
[05/Nov/2024 05:28:53] INFO autoreload Watching for file changes with StatReloader
[05/Nov/2024 05:32:50] INFO autoreload /home/dafa/gestion-recettes/Backend/recettes/views.py changed, reloading.
[05/Nov/2024 05:32:50] INFO autoreload Watching for file changes with StatReloader
[05/Nov/2024 05:33:45] INFO autoreload /home/dafa/gestion-recettes/Backend/recettes/views.py changed, reloading.
[05/Nov/2024 05:33:46] INFO autoreload Watching for file changes with StatReloader
[05/Nov/2024 05:34:07] INFO views Tentative de connexion pour l'utilisateur: lala
[05/Nov/2024 05:34:07] INFO views Données de la requête: {'username': 'lala', 'password': '1234567'}
[05/Nov/2024 05:34:07] INFO views Utilisateur lala authentifié avec succès
[05/Nov/2024 05:34:12] WARNING log Not Found: /api/current-user/
[05/Nov/2024 05:34:12] WARNING log Not Found: /api/current-user/
[05/Nov/2024 05:34:21] WARNING log Not Found: /api/current-user/
[05/Nov/2024 05:34:21] WARNING log Not Found: /api/current-user/
[05/Nov/2024 05:38:09] INFO autoreload /home/dafa/gestion-recettes/Backend/utilisateurs/views.py changed, reloading.
[05/Nov/2024 05:38:09] INFO autoreload Watching for file changes with StatReloader
[05/Nov/2024 05:38:43] INFO autoreload /home/dafa/gestion-recettes/Backend/planificateur_repas/urls.py changed, reloading.
[05/Nov/2024 05:38:44] INFO autoreload Watching for file changes with StatReloader
[05/Nov/2024 05:38:49] ERROR log Internal Server Error: /api/current-user/
Traceback (most recent call last):
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/core/handlers/exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/views/decorators/csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/views/generic/base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/Backend/utilisateurs/views.py", line 198, in get
    serializer = UtilisateurSerializer(request.user)
                 ^^^^^^^^^^^^^^^^^^^^^
NameError: name 'UtilisateurSerializer' is not defined. Did you mean: 'UtilisateurSerialiseur'?
[05/Nov/2024 05:38:49] ERROR log Internal Server Error: /api/current-user/
Traceback (most recent call last):
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/core/handlers/exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/views/decorators/csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/views/generic/base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/Backend/utilisateurs/views.py", line 198, in get
    serializer = UtilisateurSerializer(request.user)
                 ^^^^^^^^^^^^^^^^^^^^^
NameError: name 'UtilisateurSerializer' is not defined. Did you mean: 'UtilisateurSerialiseur'?
[05/Nov/2024 05:39:05] INFO views Tentative de connexion pour l'utilisateur: lala
[05/Nov/2024 05:39:05] INFO views Données de la requête: {'username': 'lala', 'password': '1234567'}
[05/Nov/2024 05:39:05] INFO views Utilisateur lala authentifié avec succès
[05/Nov/2024 05:39:09] ERROR log Internal Server Error: /api/current-user/
Traceback (most recent call last):
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/core/handlers/exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/views/decorators/csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/views/generic/base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/Backend/utilisateurs/views.py", line 198, in get
    serializer = UtilisateurSerializer(request.user)
                 ^^^^^^^^^^^^^^^^^^^^^
NameError: name 'UtilisateurSerializer' is not defined. Did you mean: 'UtilisateurSerialiseur'?
[05/Nov/2024 05:39:10] ERROR log Internal Server Error: /api/current-user/
Traceback (most recent call last):
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/core/handlers/exception.py", line 56, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/views/decorators/csrf.py", line 55, in wrapped_view
    return view_func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/django/views/generic/base.py", line 103, in view
    return self.dispatch(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/rest_framework/views.py", line 509, in dispatch
    response = self.handle_exception(exc)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/rest_framework/views.py", line 469, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/rest_framework/views.py", line 480, in raise_uncaught_exception
    raise exc
  File "/home/dafa/gestion-recettes/venv/lib/python3.12/site-packages/rest_framework/views.py", line 506, in dispatch
    response = handler(request, *args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/dafa/gestion-recettes/Backend/utilisateurs/views.py", line 198, in get
    serializer = UtilisateurSerializer(request.user)
                 ^^^^^^^^^^^^^^^^^^^^^
NameError: name 'UtilisateurSerializer' is not defined. Did you mean: 'UtilisateurSerialiseur'?
[05/Nov/2024 05:40:22] INFO autoreload /home/dafa/gestion-recettes/Backend/utilisateurs/views.py changed, reloading.
[05/Nov/2024 05:40:23] INFO autoreload Watching for file changes with StatReloader
[05/Nov/2024 05:40:32] INFO views Début de la génération du plan repas - Données reçues: {'utilisateur': 7, 'date_debut': '2024-11-11', 'date_fin': '2024-11-17'}
[05/Nov/2024 05:40:32] INFO views Paramètres : utilisateur=7, date_debut=2024-11-11, date_fin=2024-11-17
[05/Nov/2024 05:40:32] INFO views Recherche des préférences pour l'utilisateur 7
[05/Nov/2024 05:40:32] INFO views Préférences trouvées - Végétarien: False, Allergies: ['pomme']
[05/Nov/2024 05:40:32] INFO views Récupération des recettes
[05/Nov/2024 05:40:32] INFO views Nombre total de recettes trouvées : 18
[05/Nov/2024 05:40:32] INFO views Allergie 'pomme' trouvée dans la recette 'Tarte aux pommes'
[05/Nov/2024 05:40:32] INFO views Allergie 'pomme' trouvée dans la recette 'Tartiflette Savoyarde'
[05/Nov/2024 05:40:32] INFO views Nombre de recettes pour petit_dejeuner: 5
[05/Nov/2024 05:40:32] INFO views Nombre de recettes pour dejeuner: 5
[05/Nov/2024 05:40:32] INFO views Nombre de recettes pour diner: 8
[05/Nov/2024 05:40:32] INFO views Plan repas généré avec succès pour 7 jours
